sig-docs/installation/community-collaboration-for-v3.0.0.md

```diff
@@ -8,7 +8,7 @@ If you would like to contribute to community, raise a PR to add yourself as an a
 
 | Task ID | Task | Assignee (GitHub ID) | Target Date | Preferred Language | PR link |
 | --- | --- | --- | --- | --- | --- |
-| # 1 | KubeSphere on Aliyun ECS | TBD  | TBD | TBD | TBD |
+| # 1 | KubeSphere on Aliyun ECS | Howie66  | Aug.17| English | https://github.com/Howie66/community |
```

Comment:
Please update your GitHub ID with its link: [Howie66](https://github.com/Howie66/), and remove the PR link. The `PR link` means the final documentation of your Pull Request, you can fill it out when you complete it.